FormElement.inc

```diff
@@ -59,6 +59,14 @@ class FormElement implements ArrayAccess {
   protected $index;
 
   /**
+   * Holds references to the original form default value for the form
+   * element.
+   *
+   * @var ReadOnlyProtected
```

Comment:
@willtp87 and @matthewperry. The magic `__set` public method allows to re-define this value even if it's inside `protected` object, but not add new members(nor this pull is changing this behaviour), so i may be wrong, but it's strictly `readonlyProtected` out of the scope of the creator class instance. So it's just a matter of changing the var definition comment? Or do you think this has further implications?

Thanks!